File: Medical_Profile/Medret.vb
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Text

Public Class Alret
 Public status As Integer = 0
 Public message As String = Nothing
 Public token As String = ""
 Public al As ArrayList = New ArrayList()
End Class

Public Class Stret
 Public status As Integer = 0
 Public message As String = Nothing
 Public token As String = ""
 Public head As String = Nothing
 Public s1 As List(Of String) = New List(Of String)()
End Class

Public Class Mret
 Public status As Integer = 0
 Public message As String = ""
 Public body As String
End Class


Public Class Level1_Return
 Public code As Integer = 200
 Public message As String = ""
 Public Preview_practice As Boolean = False
 Public Prc As Practiceinfo_return
 Public Dpt As Dept_Return()
 Public Prv As Provider_return()
 Public Block_names As String()
 Public dsl As List(Of Dsave)
End Class

Public Class Match_Return
 Public Property Departmentid As String
 Public Property Lastname As String
 Public Property Firstname As String
 Public Property Patientid As String
 Public Property Dob As String
End Class

Public Class Level2_Return
 Public ex As Exception
 Public code As Integer = 200
 Public message As String
 Public Pat As Patient_Return
 Public mtr As List(Of Match_Return) = New List(Of Match_Return)()
 Public Ins As String = Nothing
 Public blks As Dictionary(Of String, List(Of String)) = New Dictionary(Of String, List(Of String))()
End Class

Public Class Register_Return
 Public ex As Exception
 Public code As Integer = 200
 Public message As String
 Public eval As String
End Class

Public Class Dsave_value_return
 Public ex As Exception
 Public code As Integer = 200
 Public message As String
 Public Dsave_value As String
End Class

Public Class Dsave
 Public Ukey As String
 Public Skey As String
 Public wrtim As String
 Public Name As String
 Public lwtim As String
 Public vers As String
End Class

Public Class Dsave_return
 Public ex As Exception
 Public code As Integer = 200
 Public message As String
 Public ds As List(Of Dsave)
 Public Dsave_value As String
End Class

Public Class Dsave_display
 Public Name As String
 Public Wrtim As String
 Public Skey As String
End Class

File: Medical_Profile/Medret.cs
using global::System;
using global::System.Collections;
using global::System.Collections.Generic;

namespace Medical_Profile
{
 public class Alret
 {
  public int status = 0;
  public string message = null;
  public string token = "";
  public ArrayList al = new ArrayList();
 }

 public class Stret
 {
  public int status = 0;
  public string message = null;
  public string token = "";
  public string head = null;
  public List<string> s1 = new List<string>();
 }

 public class Mret
 {
  public int status = 0;
  public string message = "";
  public string body;
 }

 public class Level1_Return
 {
  public int code = 200;
  public string message = "";
  public bool Preview_practice = false;
  public Practiceinfo_return Prc;
  public string Eval;
  public Dept_Return[] Dpt;
  public Provider_return[] Prv;
  public string[] Block_names;
  public List<Dsave> dsl;
 }

 public class Match_Return
 {
  public string Departmentid { get; set; }
  public string Lastname { get; set; }
  public string Firstname { get; set; }
  public string Patientid { get; set; }
  public string Dob { get; set; }
 }

 public class Level2_Return
 {
  public Exception ex;
  public int code = 200;
  public string message;
  public Patient_Return Pat;
  public List<Match_Return> mtr = new List<Match_Return>();
  public string Ins = null;
  public Dictionary<string, List<string>> blks = new Dictionary<string, List<string>>();
 }

 public class Register_Return
 {
  public Exception ex;
  public int code = 200;
  public string message;
  public string eval;
 }

 public class Ckup_Return
 {
  public Exception ex;
  public int code = 200;
  public string message;
  public string eval;
 }
 public class Dsave_value_return
 {
  public Exception ex;
  public int code = 200;
  public string message;
  public string Dsave_value;
 }

 public class Dsave
 {
  public string Ukey;
  public string Skey;
  public string wrtim;
  public string Name;
  public string lwtim;
  public string vers;
 }

 public class Dsave_return
 {
  public Exception ex;
  public int code = 200;
  public string message;
  public List<Dsave> ds;
  public string Dsave_value;
 }

 public class Dsave_display
 {
  public string Name;
  public string Wrtim;
  public string Skey;
 }
}